unlock-protocol.com/blog/how-we-use-openzeppelin.md

```diff
@@ -0,0 +1,65 @@
+---
+title: How we use OpenZeppelin
+subTitle: Showcasing our usage of open-source tools.
+authorName: Nick Furfaro
+publishDate: February 24, 2020
+description: Openzeppelin provides the ethereum community an excellent assortment of reusable libraries and command-line tools. Here's how we use them at Unlock.
+image: /static/images/blog/Zeplin.jpeg
```

Comment:
This is not rendering.

Reply:
hmm, will have a look. Not sure why not. The resource exists and the path is correct...

Reply:
This is the first blog post I've worked on, so I'm not sure where I should see it rendering prior to publishing?

Reply:
ah run `yarn dev` to open the site and then click blog (top right)

Reply:
Ok, got it to render.